picasso-sample/src/main/java/com/example/picasso/SampleApplication.java

```diff
@@ -0,0 +1,19 @@
+package com.example.picasso;
+
+import android.app.Application;
+import com.squareup.picasso.Picasso;
+
+// Note: Production apps should instead use the dependency injection design pattern to provide
```

Comment:
You're already using the DI pattern everywhere below this. I think you should say DI _library_ or traditional singleton pattern here.

Reply:
updated.  see `PicassoProvider.get(Context)`.